sdk/quantum/quantum-jobs/README.md

```diff
@@ -1,41 +1,209 @@
-## Azure QuantumJobClient client library for JavaScript
+## Azure Quantum Jobs client library for JavaScript
 
 This package contains an isomorphic SDK for QuantumJobClient.
 
+Azure Quantum is a Microsoft Azure service that you can use to run quantum computing programs or solve optimization problems in the cloud. Using the Azure Quantum tools and SDKs, you can create quantum programs and run them against different quantum simulators and machines. You can use the Azure.Quantum.Jobs client library to:
+
+- Create, enumerate, and cancel quantum jobs
+- Enumerate provider status and quotas
+
+  [Source code][source] | [API reference documentation](https://docs.microsoft.com/qsharp/api/) | [Product documentation](https://docs.microsoft.com/azure/quantum/)
+
 ## Getting started
 
+This section includes everything a developer needs to install and create their first client connection _very quickly_.
+
 ### Install the package
 
+Install the Azure Quantum Jobs client library for Javascript with `npm`:
+
 ```bash
 npm install @azure/quantum-jobs
 ```
 
 ### Prerequisites
 
 - Node.js version 8.x.x or higher
+- [Azure subscription](https://azure.microsoft.com/free/)
+- [Azure Quantum Workspace][workspaces]
+
+### Authenticate the client
+
+To authenticate with the service, you can use [DefaultAzureCredential](https://docs.microsoft.com/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet). This will try different authentication mechanisms based on the environment (e.g. Environment Variables, ManagedIdentity, CachedTokens) and finally it will fallback to InteractiveBrowserCredential.
```

Comment:
> InteractiveBrowserCredential

I'm sure it would work, please just do sanity testing before the release with [bundling docs](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md) since we are stating it here.

And please consider adding a section for the browser use case such as... https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/README.md#javascript-bundle